docs/build/collator/how-to-stake-in-zeitgeist.md

```diff
@@ -1,46 +1,43 @@
-# How to stake in Zeitgeist
+# How to Stake on Zeitgeist
 
-## Stake
+Head to [polkadot.js](https://https://polkadot.js.org/apps/) to control your
+account's staking.
 
-### Get the list of candidates
+### Get the List of Candidates
 
-1. Head to [Polkadot.js](https://https://polkadot.js.org/apps/)
+1. Choose **Developer** > **Chain state**
 
-2. Choose `developer`---`Chain State`
-
-3. Choose `parachainStaking`
+1. Choose `parachainStaking`
 
-4. Choose `candidatePool(): ParachainStakingSetOrderedSetBond`
+1. Choose `candidatePool(): ParachainStakingSetOrderedSetBond`
 
-5. Click `+` to make a query
+1. Click `+` to make a query
 
-6. Copy one of the canditate address you would to stake your tokens
+1. Copy the candidate address that you want to delegate your tokens to
 
 ![](/img/get-candidates-list.png)
 
 ### Get the Candidate Delegation Count
 
-Choose `developer`---`Chain State`
+1. Choose **Developer** > **Chain state**
```

Comment:
```suggestion
1. Choose **Developer** > **Chain state** > **parachainStaking**
```

Reply:
Its marked in the diagram though, we have to choose _parachainStaking_ before selecting _candidateInfo(AccountId32)_

Reply:
Thanks, I missed that!